### PR TITLE
fix classfull address naming

### PR DESCRIPTION
**Pull request by vivekvashist**

### CODE REVIEW

**Comment by abates** [comment]
I think if we're being pedantic, it should be `classful` with only one "l", see [here](https://learningnetwork.cisco.com/s/article/fundamentals-of-classful-ipv4-addressing)

**Comment by vivekvashist** [comment]
> I think if we're being pedantic, it should be `classful` with only one "l", see [here](https://learningnetwork.cisco.com/s/article/fundamentals-of-classful-ipv4-addressing)

Ah, that is technically correct - If you want I can update this RR and resubmit.

File: netutils/ip.py [hunk 1]
```diff
@@ -159,7 +159,7 @@ def ip_subtract(ip: str, val: int) -> str:
 
 
 def is_classfull(ip_network: str) -> bool:  # noqa: D300,D301
-    """Determines if a CIDR network address is within unicast class full boundaries.
+    """Determines if a CIDR network address is within unicast classfull boundaries.
```
Comment (jeffkala) [inline netutils/ip.py:162]:
```suggestion
    """Determines if a CIDR network address is within unicast classful boundaries.
```

File: netutils/ip.py [hunk 2]
```diff
@@ -171,7 +171,7 @@ def is_classfull(ip_network: str) -> bool:  # noqa: D300,D301
         ip_network: A network string that can be parsed by ipaddress.ip_network.
 
     Returns:
-        Whether or not the network falls within class full boundaries.
+        Whether or not the network falls within classfull boundaries.
```
Comment (jeffkala) [inline netutils/ip.py:174]:
```suggestion
        Whether or not the network falls within classful boundaries.
```

File: netutils/ip.py [hunk 3]
```diff
@@ -200,7 +200,7 @@ def is_classfull(ip_network: str) -> bool:  # noqa: D300,D301
           network 172.16.1.0 mask 255.255.255.0
     """
     net = ipaddress.ip_network(ip_network)
-    # Only IPv4 addresses can be classified as class full
+    # Only IPv4 addresses can be classified as classfull
```
Comment (jeffkala) [inline netutils/ip.py:203]:
```suggestion
    # Only IPv4 addresses can be classified as classful
```

**Review by jeffkala — CHANGES_REQUESTED** [review]

File: netutils/ip.py [hunk 4]
```diff
@@ -159,7 +159,7 @@ def ip_subtract(ip: str, val: int) -> str:
 
 
 def is_classfull(ip_network: str) -> bool:  # noqa: D300,D301
```
Comment (abates) [inline netutils/ip.py:161]:
```suggestion
def is_classful(ip_network: str) -> bool:  # noqa: D300,D301
```

**Comment by abates** [comment]
> Ah, that is technically correct - If you want I can update this RR and resubmit.

Since you already have a PR going those changes would be great. I still can't believe I spelled it wrong in the first place 🤦

**Comment by vivekvashist** [comment]
Apologies, I got my branching messed up and accidentally created a new PR :(

**Comment by vivekvashist** [comment]
Done.

**Comment by vivekvashist** [comment]
Not sure if this was closed by accident but the following changes are are still pending 

https://github.com/networktocode/netutils/pull/173/commits/91c777d663de655cca87ab53ef19abdd521786fe

**Comment by abates** [comment]
@vivekvashist we got confused with the other PR and thought these changes had already been merged.

**Comment by abates** [comment]
@vivekvashist can you please fix this PR so that the tests pass?

**Comment by vivekvashist** [comment]
```
(netutils-py3.10) ~/D/netutils ❯❯❯ pytest -k test_docs_generated --no-header                                              ✘ 4 develop
======================================================== test session starts ========================================================
collected 438 items / 437 deselected / 1 selected

tests/unit/test_docs.py::test_docs_generated PASSED                                                                           [100%]

================================================= 1 passed, 437 deselected in 1.89s =================================================
(netutils-py3.10) ~/D/netutils ❯❯❯
```

**Review by abates — APPROVED** [review]

**Review by itdependsnetworks — APPROVED** [review]

**Pull request merged**